imap/sync_support.c

```diff
@@ -1019,6 +1056,12 @@ int sync_sieve_delete(const char *userid __attribute__((unused)),
 {
     return 0;
 }
+
+static int sync_sieve_validate(struct index_record *record,
```

Comment:
This will need `__attribute__((unused))` on its parameters or it will fail to compile

Reply:
Actually, this stub can be removed entirely.